guides/image-sources/prov-create-host-pxe.iuml

```diff
@@ -0,0 +1,16 @@
+User -> Foreman : clicks **Create Host**
+Foreman -> Proxy : requests free IP address
+Proxy -> DHCP : reads DHCP cofiguration
+note over Foreman : pings IP address
+User -> Foreman : submits the **Create host** form
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : renders bootloader configuration files
+end
+Proxy -> TFTP : deploys bootloader configuration files
+Foreman -> Proxy : commands to download installation media
```

Comment:
> Plus there should be another step:
> 
> ```
> Proxy -> TFTP : Download files to `/var/lib/tftpboot`
> ```

That's meant by "Deploys bootloader configuration files"

Reply:
> Is this scenario with Katello or without it?
> 
> Because without Katello, we download only kernel and initram disk.

That's happening in a different place. See [this line](#diff-85e13b43df4a63adcfc90d46884dcbd1ac9b7f171a1efb665c1f84f0c6a3c8abR32)

Reply:
> Download of kernel and initram disk is happening in a different place. See [this line](https://github.com/theforeman/foreman-documentation/pull/3304/files#diff-85e13b43df4a63adcfc90d46884dcbd1ac9b7f171a1efb665c1f84f0c6a3c8abR32)

Yes, but that's a second step. When you create a host, we download the kernel and init the RAMdisk to the smart proxy first.
From there, the machine will download and run it.